src/__tests__/e2e/homepage_test.py

```diff
@@ -39,32 +49,27 @@ def test_h1_exists(self):
         """Test if the h1 element is displayed"""
         h1_rendered = self.driver.find_element(By.TAG_NAME, 'h1')
         self.assertTrue(h1_rendered.is_displayed())
-        self.assertEqual(h1_rendered.text,
-                         'GET 8 AAPI FRIENDS TO REGISTER TO VOTE IN 8 DAYS')
+        self.assertEqualIgnoreCase('GET 8 AAPI FRIENDS TO REGISTER TO VOTE IN 8 DAYS', h1_rendered.text)
 
     def test_locate_button(self):
         """Test if the challenge button is displayed correctly"""
         button_element = self.driver.find_element(
             By.CSS_SELECTOR, ".styles_challenge_btn__T0eGB")
-        if button_element.text == 'TAKE THE CHALLENGE':
-            self.assertEqual(button_element.text, 'TAKE THE CHALLENGE')
-        else:
-            self.fail(
-                f"expected texted TAKE THE CHALLENGE but got {button_element.text}")
+        self.assertEqualIgnoreCase('TAKE THE CHALLENGE', button_element.text)
             
-# HamBurger Menu    
+    # Hamburger Menu    
     def test_hamburger_menu(self):
         """Test if the hambuger menu functions as intended"""
-        prevState = self.driver.find_element(By.CLASS_NAME, 'hidden').is_selected()
-        if not prevState:
-            alterState = self.driver.find_element(By.CLASS_NAME, 'styles_outer_container__ppLtJ')
+        prev_state = self.driver.find_element(By.CLASS_NAME, 'hidden').is_selected()
+        if not prev_state:
+            alter_state = self.driver.find_element(By.CLASS_NAME, 'styles_outer_container__ppLtJ')
             events = ActionChains(self.driver)
-            events.move_to_element(alterState).click().perform()
+            events.move_to_element(alter_state).click().perform()
         else:
             self.fail('Hamburger Menu Tests have failed!')
     
-        innerMenu = self.driver.find_element(By.CLASS_NAME, 'styles_inner_container__0JSHj').is_displayed()
-        if innerMenu:
+        inner_menu = self.driver.find_element(By.CLASS_NAME, 'styles_inner_container__0JSHj').is_displayed()
+        if inner_menu:
             return None
```

Comment:
Is there a reason this is returning None? Could this instead check **if not inner_menu** and get rid of the else?

Reply:
Yes, this can probably be simplified. Thanks!